app/views/mountain_view/styleguide/show.html.erb

```diff
@@ -2,24 +2,46 @@
   <h1><%= @component.title %></h1>
 </div>
 
-<% if @component.stubs? %>
-  <% @component.styleguide_stubs.each_with_index do |component_properties, index| %>
-    <div class="mv-component">
-      <div class="mv-component__item">
-        <%= render_component @component.name, component_properties %>
-      </div>
-      <div class="mv-component__description">
-        <h2><%= @component.title %> <%= index + 1 %></h2>
-        <div class="mv-component__description__definition">
-          <code class="language-ruby">&lt;%= render_component("<%= @component.name %>", {properties as below}) %&gt;</code>
+<% if @component.stubs? && @component.stubs_correct_format? %>
+  <% if @component.stubs_extra_info? %>
+    <div class="mv-component__meta">
+      <p><%= @component.stubs_extra_info %></p>
+    </div>
+  <% end %>
+  <% if @component.component_stubs? %>
+    <% @component.component_stubs.each_with_index do |component_properties, index| %>
+        <div class="mv-component">
+          <div class="mv-component__item">
+            <%= render_component @component.name, component_properties %>
+          </div>
+          <div class="mv-component__description">
+            <h2><%= @component.title %> <%= index + 1 %></h2>
+            <div class="mv-component__description__definition">
+              <code class="language-ruby">&lt;%= render_component("<%= @component.name %>", {properties as below}) %&gt;</code>
+            </div>
+            <%- formatted = JSON.pretty_generate component_properties %>
+            <div class="mv-component__description__properties">
+              <code class="language-ruby"><%= formatted.gsub(/\"(\S+)?\":/, '\1:') %></code>
+            </div>
+          </div>
         </div>
-        <%- formatted = JSON.pretty_generate component_properties %>
+    <% end %>
+  <% end %>
+<% elsif @component.stubs? && !@component.stubs_correct_format? %>
+  <div class="mv-hint">
```

Comment:
I have no idea it was that causing the extra space. I will change that ASAP so we can close this issue.

Thanks @jgnatch.
Also I will work on the way of adding more pages for the styleguide. :+1: